webpack.base.config.js

```diff
@@ -30,6 +30,9 @@ module.exports = {
         new Dotenv(),
         new webpack.IgnorePlugin({
             resourceRegExp: /\@chainsafe\/blst/,
-          })
+          }),
+          new webpack.DefinePlugin({
+            "process.env.DOCKER_LIGHTHOUSE_IMAGE": JSON.stringify(process.env.DOCKER_LIGHTHOUSE_IMAGE || "sigp/lighthouse:v1.0.5")
```

Comment:
better use `sigp/lighthouse:latest` verzion `1.0.5` could be deprecated in few months

Reply:
ha, double edged sword :D
We can let users do "docker pull" if update is needed. To me looks better to be on stable version that's working for us